Resources/Prototypes/_Harmony/Catalog/Fills/Crates/armory.yml

```
@@ -44,3 +44,27 @@
       - id: SpeedLoaderLightRifle
         orGroup: ammo
         amount: 42
+
+- type: entity
+  id: CrateArmorySMGHarmony
+  suffix: Drozd
```

The crate should also keep the secure suffix
```suggestion
  suffix: Drozd, Secure
```